src/pykeen/nn/modules.py

```diff
@@ -765,7 +765,7 @@ def __init__(
             the number of filters (=output channels) of the convolution
         """
         super().__init__()
-        self.embedding_dim = embedding_dim
+        self.embedding_dim = embedding_dim  # FIXME rename
```

Comment:
there are more of them, e.g., ConvE.

In some places it makes sense when those interactions expect a vector as representation (which is true for the majority of them)